app/services/foreman_ansible/operating_system_parser.rb

```diff
@@ -68,8 +70,12 @@ def os_release
     end
 
     def os_minor
-      _, minor = os_release&.split('.', 2) ||
+      if os_name == 'Ubuntu'
+        minor = nil
```

Comment:
Ubuntu does have minor releases. There's `22.04.2` now for example. I don't know if this matters in things like URLs to repositories. You do track it for Puppet so things may flip-flop.